packages/@sanity/desk-tool/src/DeskToolPanes.js

```diff
@@ -98,7 +104,7 @@ export default class DeskToolPanes extends React.Component {
   componentDidUpdate(prevProps) {
     if (this.props.panes.length !== prevProps.panes.length) {
       this.userCollapsedPanes = []
-      this.handleAutoCollapse(this.state.windowWidth, undefined, this.userCollapsedPanes)
+      this.handleAutoCollapse(this.state.width, undefined, this.userCollapsedPanes)
```

Comment:
What's the difference between `width` and `windowWidth` here? Do we still need both?

Reply:
I don't know. Perhaps @jorngeorg or @vicbergquist can chip in here?